### PR TITLE
Task #198196  AG - form UI

### PR DESCRIPTION
**Pull request by Ekta29yadav**

### CODE REVIEW

File: apps/front-end/src/pages/front-end/ag-form/Docschecklist.js [hunk 1]
```diff
@@ -121,23 +109,17 @@ const Docschecklist = () => {
...
           alignItems={"center"}
           justifyContent={"space-between"}
         >
-          <Text>{t("MOBILE_NUMBER")}</Text>
-          <Select
-            borderWidth={"0px"}
-            borderBottomWidth={"2px"}
-            borderRadius={"0"}
-            borderColor={"black"}
+          <Text fontSize="sm" bold color="textMaroonColor.400">{t("MOBILE_NUMBER")}</Text>
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/ag-form/Docschecklist.js:120]: use typo component

File: apps/front-end/src/pages/front-end/ag-form/Docschecklist.js [hunk 2]
```diff
@@ -150,23 +132,17 @@ const Docschecklist = () => {
...
           alignItems={"center"}
           justifyContent={"space-between"}
         >
-          <Text>{t("MARKSHEET")}</Text>
-          <Select
-            borderWidth={"0px"}
-            borderBottomWidth={"2px"}
-            borderRadius={"0"}
-            borderColor={"black"}
+          <Text fontSize="sm" bold color="textMaroonColor.400">{t("MARKSHEET")}</Text>
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/ag-form/Docschecklist.js:143]: use typo component

File: apps/front-end/src/pages/front-end/ag-form/Docschecklist.js [hunk 3]
```diff
@@ -179,23 +155,17 @@ const Docschecklist = () => {
...
           alignItems={"center"}
           justifyContent={"space-between"}
         >
-          <Text>{t("BANK_PASSBOOK")}</Text>
-          <Select
-            borderWidth={"0px"}
-            borderBottomWidth={"2px"}
-            borderRadius={"0"}
-            borderColor={"black"}
+          <Text fontSize="sm" bold color="textMaroonColor.400">{t("BANK_PASSBOOK")}</Text>
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/ag-form/Docschecklist.js:166]: use typo component

File: apps/front-end/src/pages/front-end/ag-form/Docschecklist.js [hunk 4]
```diff
@@ -208,23 +178,17 @@ const Docschecklist = () => {
...
           alignItems={"center"}
           justifyContent={"space-between"}
         >
-          <Text>{t("BIRTH_CERTIFICATE")}</Text>
-          <Select
-            borderWidth={"0px"}
-            borderBottomWidth={"2px"}
-            borderRadius={"0"}
-            borderColor={"black"}
+          <Text fontSize="sm" bold color="textMaroonColor.400">{t("BIRTH_CERTIFICATE")}</Text>
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/ag-form/Docschecklist.js:189]: use typo component

File: apps/front-end/src/pages/front-end/ag-form/LearnerProfile.js [hunk 5]
```diff
@@ -84,64 +78,58 @@ const LearnerProfile = () => {
...
-            height="15px"
-          />
+        <VStack>
+        <IconByName name="AccountCircleLineIcon" color='gray.300' _icon={{ size: "120px" }} />
         </VStack>
 
-        <VStack width={"100%"} mt={3}>
-          <Text ml={3} fontSize={"12px"}>
+        <VStack width="100%" pl="4">
+          <Text fontSize="sm" color="textGreyColor.600">
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/ag-form/LearnerProfile.js:86]: use language constant and typo component

File: apps/front-end/src/pages/front-end/ag-form/LearnerProfile.js [hunk 6]
```diff
@@ -84,64 +78,58 @@ const LearnerProfile = () => {
...
           </Text>
         </VStack>
 
         <VStack width={"100%"}>
-          <Collapsible header={t("PROFILE_DETAILS")}>
-            <VStack mt={4} borderBottomWidth={"1px"} borderColor={"#666666"}>
-              <Text>Basic</Text>
+          <Collapsible fontSize="sm" color="textGreyColor.600" bold header={t("PROFILE_DETAILS")}>
+            <VStack mt={4} borderBottomWidth={"1px"} borderColor="textGreyColor.600">
+              <Text fontSize="sm" color="textGreyColor.600">Basic</Text>
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/ag-form/LearnerProfile.js:94]: use language constant and typo component

File: apps/front-end/src/pages/front-end/ag-form/LearnerProfile.js [hunk 7]
```diff
@@ -84,64 +78,58 @@ const LearnerProfile = () => {
...
             </VStack>
-            <VStack mt={4} borderBottomWidth={"1px"} borderColor={"#666666"}>
-              <Text>Educational</Text>
+            <VStack mt={4} borderBottomWidth={"1px"} borderColor="textGreyColor.600">
+              <Text fontSize="sm" color="textGreyColor.600">Educational</Text>
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/ag-form/LearnerProfile.js:97]: use language constant and typo component

File: apps/front-end/src/pages/front-end/ag-form/LearnerProfile.js [hunk 8]
```diff
@@ -84,64 +78,58 @@ const LearnerProfile = () => {
...
             </VStack>
-            <VStack mt={4} borderBottomWidth={"1px"} borderColor={"#666666"}>
-              <Text>Aadhaar</Text>
+            <VStack mt={4} borderBottomWidth={"1px"} borderColor="textGreyColor.600">
+              <Text fontSize="sm" color="textGreyColor.600">Aadhaar</Text>
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/ag-form/LearnerProfile.js:100]: use language constant and typo component

File: apps/front-end/src/pages/front-end/ag-form/LearnerProfile.js [hunk 9]
```diff
@@ -84,64 +78,58 @@ const LearnerProfile = () => {
...
+              <Text fontSize="sm" color="textGreyColor.600">Basic</Text>
             </VStack>
             <VStack mt={4} borderBottomWidth={"1px"} borderColor={"#666666"}>
-              <Text>Educational</Text>
+              <Text fontSize="sm" color="textGreyColor.600">Educational</Text>
             </VStack>
             <VStack mt={4} borderBottomWidth={"1px"} borderColor={"#666666"}>
-              <Text>Aadhaar</Text>
+              <Text fontSize="sm" color="textGreyColor.600">Aadhaar</Text>
             </VStack>
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/ag-form/LearnerProfile.js:119]: use language constant and typo component

File: apps/front-end/src/pages/front-end/ag-form/LearnerProfile.js [hunk 10]
```diff
@@ -84,64 +78,58 @@ const LearnerProfile = () => {
...
-              <Text>Basic</Text>
+              <Text fontSize="sm" color="textGreyColor.600">Basic</Text>
             </VStack>
             <VStack mt={4} borderBottomWidth={"1px"} borderColor={"#666666"}>
-              <Text>Educational</Text>
+              <Text fontSize="sm" color="textGreyColor.600">Educational</Text>
             </VStack>
             <VStack mt={4} borderBottomWidth={"1px"} borderColor={"#666666"}>
-              <Text>Aadhaar</Text>
+              <Text fontSize="sm" color="textGreyColor.600">Aadhaar</Text>
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/ag-form/LearnerProfile.js:132]: use language constant and typo component

**Comment by sagarkoshti1990** [comment]
Please run yarn format at project root path.

File: apps/front-end/src/pages/admin/facilitator/View.js [hunk 11]
```diff
@@ -185,7 +193,7 @@ export default function FacilitatorView({ footerLinks }) {
                     name="CellphoneLineIcon"
                     color="textGreyColor.300"
                   />
-                  <BodyLarge>{data?.mobile}</BodyLarge>
+                  <Text>{data?.mobile}</Text>
```
[the remark this answers is not in the document]
Comment (Ekta29yadav) [reply apps/front-end/src/pages/admin/facilitator/View.js:196]: solved this in recent PR

File: apps/front-end/src/pages/admin/facilitator/Table.js [hunk 12]
```diff
@@ -252,9 +148,7 @@ function Table({ facilitator, setadminPage, setadminLimit, admindata }) {
                   navigate(`/admin/view/${row?.id}`);
                 }}
               >
-                <Text fontSize="12px" color={"white"}>
-                  {t("VIEW")}
-                </Text>
+               <Text fontSize="12px" color={"white"}> {t("VIEW")}</Text>
```
[the remark this answers is not in the document]
Comment (Ekta29yadav) [reply apps/front-end/src/pages/admin/facilitator/Table.js]: done

File: apps/front-end/src/pages/front-end/AgLearnerProfileView.js [hunk 13]
```diff
@@ -217,25 +217,15 @@ export default function AgLearnerProfileView() {
               </HStack>
             </VStack>
           </Box>
-          <Button
-            bgColor="white"
-            borderColor="#790000"
+          <FrontEndTypo.Secondarybutton
             borderRadius="100px"
```
[the remark this answers is not in the document]
Comment (Ekta29yadav) [reply apps/front-end/src/pages/front-end/AgLearnerProfileView.js]: done

File: apps/front-end/src/pages/front-end/ag-form/Agduplicate.js [hunk 14]
```diff
@@ -49,11 +49,10 @@ const Agduplicate = () => {
...
           <HStack width={"90%"} mx={"auto"} alignItems={"center"}>
-            <IconByName name="ErrorWarningLineIcon" />
-
-            <VStack>
-              <Text ml={3}>{t("AG_LEARNER_ALREADY_IDENTIFIED")}</Text>
-              <Text ml={3} fontSize={10} width={"80%"}>
+            <IconByName name="ErrorWarningLineIcon" color="textRed.300" size="20px" />
+            <VStack pl="3">
+              <FrontEndTypo.H2  color="textGreyColor.600">{t("AG_LEARNER_ALREADY_IDENTIFIED")}</FrontEndTypo.H2>
+              <Text fontSize="xs" color="textGreyColor.600">
```
[the remark this answers is not in the document]
Comment (Ekta29yadav) [reply apps/front-end/src/pages/front-end/ag-form/Agduplicate.js]: done

File: apps/front-end/src/pages/front-end/ag-form/Agduplicate.js [hunk 15]
```diff
@@ -75,168 +74,123 @@ const Agduplicate = () => {
...
-                    height="50px"
-                  />
-                  <VStack>
-                    <Text mt={3} fontSize={16}>
+        <Modal isOpen={addmodal} onClose={() => setaddmodal(false)} size="md">
+          <Modal.Content py="0">
+            <Modal.Body >
+              <VStack alignItems={"center"}>
+                  <IconByName name="CheckboxCircleLineIcon" color="textGreyColor.150" _icon={{ size: "50px" }}></IconByName>
+                    <Text fontSize="xl" pb="2" color="worksheetBoxText.400" bold>
```
[the remark this answers is not in the document]
Comment (Ekta29yadav) [reply apps/front-end/src/pages/front-end/ag-form/Agduplicate.js]: done

File: apps/front-end/src/pages/front-end/ag-form/LearnerProfile.js (hunk 6 above)
Comment (Ekta29yadav) [reply apps/front-end/src/pages/front-end/ag-form/LearnerProfile.js]: done

File: apps/front-end/src/pages/front-end/ag-form/LearnerProfile.js (hunk 7 above)
Comment (Ekta29yadav) [reply apps/front-end/src/pages/front-end/ag-form/LearnerProfile.js]: done

File: apps/front-end/src/pages/front-end/ag-form/LearnerProfile.js (hunk 10 above)
Comment (Ekta29yadav) [reply apps/front-end/src/pages/front-end/ag-form/LearnerProfile.js]: done

File: apps/front-end/src/pages/front-end/ag-form/LearnerProfile.js (hunk 9 above)
Comment (Ekta29yadav) [reply apps/front-end/src/pages/front-end/ag-form/LearnerProfile.js]: done

File: apps/front-end/src/pages/front-end/ag-form/LearnerProfile.js (hunk 8 above)
Comment (Ekta29yadav) [reply apps/front-end/src/pages/front-end/ag-form/LearnerProfile.js]: done

**Pull request merged**